### DISCUSSION TITLE
Streams substantially delayed

### DISCUSSION THREAD

**Discussion by blistovmhz** [mrlt8/docker-wyze-bridge#1069] [Q&A]
Noticed this with quite a few other shitty cameras RTSP recently. I'm not new to cameras, but been out of it for a while, and man, cameras rtsp didn't used to be delayed by several seconds 20 years ago :p.
Saw this in the bridge as well. The shitty wyze app (is super shitty) shows maybe 500ms delay, while the bridge is showing between 5 and 50 seconds delay. Why is this?

**Comment by mrlt8** [comment]
Do you have audio enabled? There's a bug in the current version that causes the video/audio to lag if audio is enabled.

**Reply by tdx3000** [reply]
-I've tried changing the router setting suggested in some other post: no change
-change video quality to low still same.
-NET_MODE set to LAN or P2P same result

as of right now the mobile app is on 5g connection and it seeing a 400, 500ms delay, while the bridge is on the same LAN seeing about 10s delay.

is there anything else to reduce the delay? Is there a variable setting to pass thru the video without encoding (if that help?)

here is my config file

version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554 # RTSP
            - 8888:8888 # HLS
            - 8889:8889 #WebRTC
            - 8189:8189/udp # WebRTC/ICE
            - 5000:5000 # WEB-UI
        environment:
            - WYZE_EMAIL=
            - WYZE_PASSWORD=
            # [OPTIONAL] IP Address of the host to enable WebRTC e.g.,:
            - WB_IP=192.168.1.241
            - FILTER_NAMES=Front Yard
            - NET_MODE=LAN
            #- ENABLE_AUDIO=True
            - SUBSTREAM=True
            - ON_DEMAND=False
            - IGNORE_OFFLINE=True
            - FPS_FIX=True
            - QUALITY=HD120

**Comment by tdx3000** [comment]
I do not have audio enable, net_mode LAN but the video is always 15 to 20s behind the wyze app. Even when load the browser after the wyze app the time stamp is way behind.
